gsoc/students-guideline.md

```diff
@@ -12,16 +12,20 @@ Get started checking out the [Google Summer of Code Guides for students](https:/
 
 ### How to apply to CERN-HSF
 
-1. Take a look at the [list of projects](/activities/gsoc.html#projects-in-2019) participating for this year.
+1. Take a look at the [list of projects](/activities/gsoc.html#projects-in-2020) participating for this year.
   * There you will find a list of proposals for each project, their descriptions and the contact information of their mentors.
-  * There is also a list with [all the project proposals](http://hepsoftwarefoundation.org/gsoc/2019/summary.html) sorted in alphabetical order.
+  * There is also a list with [all the project proposals](http://hepsoftwarefoundation.org/gsoc/2020/summary.html) sorted in alphabetical order.
   <!-- Next <br><br>, add an extra new line, otherwise there is no space between point 2 and previous nested bullet  -->
   <br><br>
-2. Once you have chosen one or more proposals that you like, please contact the corresponding mentors and they will guide you through the next steps.
+2. Once you have chosen one or more proposals that you like, please contact the corresponding mentors (their emails can be found at the end of each proposal) and they will guide you through the next steps.
 <!-- Next <br><br>, add an extra new line  -->
 <br><br>
 3. Mentors will typically ask you to complete a technical test to evaluate if you have the skills required for the project.
   * Please note that the vast majority of our GSoC projects do not require any physics knowledge; in case of doubt, ask the mentors.
   * If you do well in the tests, mentors will usually ask you to write a description for your proposal and help you with the submission.
 
-Do not hesitate to [contact us](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com) if you have further questions.
+In case you want to express interest in several projects you can also write to the general HSF GSoC mentors mailing list (including also previous years mentors). Note however that you may not be contacted back unless to adress directly to the mentors of specific projects.
```

Comment:
"In case" -> "In the case that" (or just "If"!)

"HSF GSoC mentors mailing list" -> "HSF GSoC mentors' mailing list"

"previous years mentors" -> "previous years' mentors"

"Note however that you may not be contacted back unless to adress directly to the mentors of specific projects." -> "However, you may not get a direct response unless you specifically write to the mentors of the projects that interest you."